### ISSUE TITLE
Explore the possiblity of importing script from CWD

### ISSUE THREAD

**Issue by noway** [piqueserver/piqueserver#52]
Actually, just had a thought about it. What if instead of changing sys.path, change the `.`? I.e. the the current working directory. That's something I haven't thought about, but it might actually work. Hopefully python allows cd'ing into python's egg content. Maybe not, but worth a try.

Regarding how modules are imported

**Comment by swalladge** [comment]
This issue would be fixed in PR #57

**Comment by noway421** [comment]
It would have been fixed if we would get rid of old imports script compatibility. But we decided not to, for now.

More explanation regarding this issue's proposal: 
1. When piqueserver is running, the CWD should be 
 `$piqueserver_egg_location/piqueserver.egg/feature_server` (YES, inside the egg, hopefully that's possible right?)
2. Then we have "." as the top priority entry in `sys.path`

That would allow us to keep sys.path sane: no crazy directories inside. The crazy side would shift into another domain: the CWD we are cd'ed in.
